Shippo/APIResource.cs

```diff
@@ -433,6 +438,49 @@ public ShippoCollection<Manifest> AllManifests (Hashtable parameters)
 
         #endregion
 
+        #region Batch
+
+        public Batch CreateBatch (String carrierAccount, String servicelevelToken, String labelFiletype,
+                                  String metadata, List<Batch.BatchShipment> batchShipments)
+        {
+            string ep = String.Format ("{0}/batches", api_endpoint);
+            Hashtable parameters = new Hashtable ();
+            parameters.Add ("default_carrier_account", carrierAccount);
+            parameters.Add ("default_servicelevel_token", servicelevelToken);
+            parameters.Add ("label_filetype", labelFiletype);
+            parameters.Add ("metadata", metadata);
+            parameters.Add ("batch_shipments", batchShipments);
+            return DoRequest<Batch> (ep, "POST", serialize(parameters));
+        }
+
+        public Batch RetrieveBatch (String id, Hashtable parameters = null)
```

Comment:
Will be nice to take `page` and `objectResults` as explicit arguments instead of `Hashtable`. See [java example](https://github.com/goshippo/shippo-java-client/blob/93911beed4b637ffe071f82c1ac3274713aa7fb8/src/main/java/com/shippo/model/Batch.java#L218) for reference.